src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -454,4 +465,29 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
       () => {}
     );
   }
+
+   // For Updating the value of Leaderboard precision
+   updateChallengephaseSplit(event: MatSliderChange) {
```

Comment:
Can you be please specific with the function name as we are only updating leaderboard precision value here? Something like `updateLeaderboardDecimalPrecision`

And also change the above comment to `Update leaderboard decimal precision value`